imagersite/imager_images/views.py

```diff
@@ -1,3 +1,86 @@
 from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.http import HttpResponse
+from .models import Photo, Album
+from django.views.generic.edit import ModelFormMixin, ProcessFormView
+from django.views.generic.detail import SingleObjectTemplateResponseMixin
+import datetime
 
-# Create your views here.
+LIBRARY_TEMPLATE = 'library.html'
+
+
+@login_required(login_url='/login')
+def Library(request, *args, **kwargs):
+    """Authenticated User Profile."""
+    albums = request.user.albums.all()
+    photos = request.user.photos.all()
+    return render(
+        request,
+        LIBRARY_TEMPLATE,
+        context={'albums': albums, 'photos': photos})
+
+
+def photo_view(request, **kwargs):
+    photo_id = kwargs.get('photo_id')
+    user = User.objects.filter(id=kwargs.get('user_id')).first()
+    image = user.photos.filter(id=photo_id).first()
```

Comment:
I'm not convinced that you need to query the db for the user in this case.  For example, filtering the photos for those that belong to the user could look like this:

```python
try:
    image = Photo.objects.filter(id=photo_id, user__pk=user_id).get()
except Photo.DoesNotExist:
    return HttpResponseNotFound()
```